hackclub.com.yaml

```
@@ -2465,10 +2465,16 @@ hupy:
     type: CNAME
     value: hupy.github.io.
 
-
+codethetech:
+  ttl: 1
+  type: CNAME
+  values: codethetech.github.io
+  domain: codethetech.hackclub.com
```

Octodns expects a `value` key, here's a quick fix!

```suggestion
  value: codethetech.github.io
```